json-definitions/v3/tech-record/enums/vehicleConfiguration.enum.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I agree there should be an Enum for Vehicle Configuration that encapsulates all of the options, regardless of vehicleTyoe, out of curiosity is there a difference between this enum and the `vehicleConfigurationLightVehicle` enum?